ui/js/filters.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+let numeral = require('numeral');
```

Comment:
This isn't the greatest, since I think it puts numeral on the global scope which could cause problems for people later. Looking at the source, it looks like we created an angular "dependency" for jsyaml. Could you do the same? Should just be a few lines of code.

https://github.com/mozilla/treeherder/blob/7633331597f7b28f962fe53ce6a3aa2ba28c7e32/ui/js/services/main.js#L183